cmd/teleirc.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ritlug/teleirc/internal"
+)
+
+func main() {
+	settings, err := internal.LoadConfig("../env.example")
```

Comment:
This is late, but if I read this right, is it loading the `env.example` file itself as the config file? What if the user has a non-default config, e.g. `.env`? Maybe I'm not reading this right?

Reply:
This isn't in the code anymore :upside_down_face: